FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/api/DomainOrgRulesResponse.kt

```diff
@@ -24,7 +24,7 @@ import com.google.gson.annotations.SerializedName
 data class DomainOrgRulesResponse constructor(
   @SerializedName("error")
   @Expose override val apiError: ApiError? = null,
-  @SerializedName("domain_org_rules")
+  @SerializedName("domain_org_rules", alternate = ["clientConfiguration"])
```

Comment:
I recommend to create separate models. Backend and FES endpoints may look similar, but their stucture will often be different.